abel/tests/test_tools.py

```diff
@@ -65,20 +63,22 @@ def test_centering_function():
         assert_equal( is_symmetric(res), True,\
             'Validating the centering function for ni={}, n_c={}'.format(ni, n_c))
 
+
 def test_speeds_non_integer_center():  
     # ensures that the rest speeds function can work with a non-integer center
     n  = 101
     IM = np.random.randn(n, n)
     abel.tools.vmi.angular_integration(IM, origin=(50.5, 50.5))
 
+
 def test_anisotropy_parameter():
     # anisotropy parameter from test image (not transformed)
     IM = abel.tools.analytical.sample_image(name='dribinski')
     
     Beta, Amp, Rmid, Ivstheta, theta = abel.tools.vmi.radial_integration(IM,
                                          radial_ranges=([(0, 33), (92, 108)]))
 
-    assert_almost_equal(-0.13, Beta[0][0], decimal=2)
+    assert_almost_equal(-0.14, Beta[0][0], decimal=2)
```

Comment:
I believe this is related to a more correctly defined center, that then yields a more negative anisotropy parameter.  beta = -0.14 cf beta = -0.13 is not a big concern.